### PR TITLE
🐛 amp-subscriptions: In viewer platform, throw error if entitlements request times out

### PR DESCRIPTION
**Pull request by ChrisAntaki**
This PR updates the viewer platform to throw an error if the entitlements request times out. The newly exported timeout duration from `amp-subscriptions.js` is used.

### CODE REVIEW

**Comment by jpettitt** [comment]
Please move constants from score-factors.js to the constants file too

**Comment by ChrisAntaki** [comment]
Good catch! They have been moved

File: extensions/amp-subscriptions/0.1/test/test-viewer-platform.js [hunk 1]
```diff
@@ -140,6 +143,17 @@ describes.fakeWin('ViewerSubscriptionPlatform', {amp: true}, env => {
...
     });
 
+    it('should throw error if entitlements request times out', async () => {
+      viewerPlatform.viewer_.sendMessageAwaitResponse.restore();
+      env.sandbox
+        .stub(viewerPlatform.viewer_, 'sendMessageAwaitResponse')
+        .callsFake(() => new Promise(() => {}));
+
+      const entitlementsPromise = viewerPlatform.getEntitlements();
+      clock.tick(6000);
```
Comment (jpettitt) [inline extensions/amp-subscriptions/0.1/test/test-viewer-platform.js]: Nit: 6000 -> ENTITLEMENTS_REQUEST_TIMEOUT + 1000

Comment (ChrisAntaki) [reply extensions/amp-subscriptions/0.1/test/test-viewer-platform.js]: Updated!

**Pull request merged**